src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -99,7 +98,7 @@ public int getWidth() {
 	 * @return the height of the field.
 	 */
 	public int getHeigth() {
-		return WINDOW_Y;
+		return (int) settings.getDouble("SCREEN_HEIGHT") - 50;
```

Comment:
Why a double? The method should return an int, so either there should be an integer setting, or the method should be changed to return a double.

<feature